eval/nei2005Config.toml

```diff
@@ -34,6 +34,9 @@ HTTPAddress = ":8080"
 # Path to desired output shapefile location. Can include environment variables.
 OutputFile = "nei2005/2005nei_output_${InMAPRunType}.shp"
 
+# Path to desired logfile location. Can include environment variables.
```

Comment:
As a general rule, we want documentation comments to be in full sentences starting with, for example, "LogFile specifies...". We should add that if LogFile is left blank, the file location will default to OutputFile with a ".log" extension instead of ".txt". We should also make sure that the program is doing that.